spacy/gold.pyx

```diff
@@ -68,9 +68,9 @@ def merge_sents(sents):
         m_deps[5].extend(ner)
         m_brackets.extend((b["first"] + i, b["last"] + i, b["label"])
                           for b in brackets)
+        m_cats.update(cats)
```

Comment:
This makes me nervous because it relies on every sentence being labeled the same way in an intermediate step or there will unexpected bugs.